packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.styles.ts

```diff
@@ -11,6 +11,8 @@ import {
   getStyles as getSplitButtonStyles
 } from '../SplitButton/SplitButton.styles';
 
+const MS_HIGHCONTRAST_ACTIVE = '@media screen and (-ms-high-contrast: active)';
```

Comment:
I'm not sure why we would do this. Applying this to root works and adding the & makes it not work. Can you clarify if I'm missing something here?